test/completion.js

```diff
@@ -38,4 +38,17 @@ describe('flag: --completion', function() {
     }
   });
 
+  it('shows error message for missing completion type', function(done) {
+    runner({ verbose: false })
+      .gulp('--completion')
+      .run(cb);
+
+    function cb(err) {
+      expect(function() {
+        throw err;
```

Comment:
The thing is if I test it with `expect(err).toMatch(/Missing completion type/)` I can type in any error message, it will still match the thrown error:  `expect(err).toMatch(/Any error/)` would still pass.

If I throw `err` inside the function I can catch the error message with `.toThrow`.